cves/CVE-2009-2902.yml

```diff
@@ -43,15 +43,21 @@ description_instructions: |
   that outsiders to Chromium would not understand. Technology like "regular
   expressions" is fine, and security phrases like "invalid write" are fine to
   keep too.
-description:
+description: This vulnerability allows a user over a network to create a specially
+  crafted WAR file that could be deployed by an authorized user. The WAR could
+  potentially overwrite files on the system running Apache Tomcat under the
+  "work/<engine-name>/<hostname>" directory. This vulnerability is an example of
+  improper limitation of a pathname to a restricted directory because a user without
+  access to the system can violate the integrity of files they should not have access
+  to.
 
 bounty_instructions: |
   If you came across any indications that a bounty was paid out for this
   vulnerability, fill it out here. Or correct it if the information already here
   was wrong. Otherwise, leave it blank.
 bounty:
   amt:
-  announced: 2010-01-28
+  announced:
```

Comment:
Is there a reason this was removed?

Reply:
I could not find any information about a bounty for this vulnerability, specifically one with this date, so I removed the date assuming it was incorrect.